migrace/018.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+$this->q("
+
+UPDATE akce_seznam SET typ=13 where typ=14;
+DELETE FROM akce_typy where id_typu=14;
+UPDATE akce_typy SET typ_1pmn='Parcon a přednášky',typ_1p='Parcon nebo přednáška' WHERE id_typu=3;
```

Comment:
Je to spíš jedno (byl to záměr), ale když už jsem byl v tom, tak jsem to opravil.